src/pages/en/faq/index.md

```diff
@@ -6,18 +6,110 @@ If this FAQ cannot solve your problem, please contact the [Technical Support Des
+### Q. What is ECCS Cloud Email?
+A.  
+* Simply put, it is UTokyo’s version of Google’s email service.
+* As ECCS Cloud Email works as a Google account, you can use Google tools and services such as
+* Google Drive (file storage service)
+* Google Docs (word processor)
+* Google Sheets (spreadsheet program)
+* Google Forms (survey administration software)
+* YouTube
+
+For more information, please refer to [“ECCS Cloud Email (Google Workspace)”](/en/google/)
+
+
+### Q. How can I start using ECCS Cloud Email?
+* Please read “Initial Setup Procedures” on the [“ECCS Cloud Email (Google Workspace)”](/en/google/)
+
+### Q. I cannot access Google Drive and files exclusive to UTokyo. What can I do?
+* (On PC or smartphone) If you are taken to the “You need access” page, and it shows you are signed in with an email address other than your `@g.ecc.u-tokyo.ac.jp` email address:
+* You are already signed into Google with a different email address. To switch to your ECCS Cloud Email, click on the email address under “You’re signed in as”. After you are taken to the “Choose an account” page, select your ECCS Cloud Email address (`xxx@g.ecc.u-tokyo.ac.jp`). If your ECCS Cloud Email address does not appear in the choices, click on “Use another account” and enter your ECCS Cloud Email address to sign in.
+
+* (On your smartphone) If you are taken to pages showing “You need permission” or “Request access”:
+  * This can happen when you are already signed in with another Google account. Follow the procedures below, and try accessing the drive/file again.  
+    1. Open a Google App such as Gmail, and press the icon in the upper right corner (either your own icon, or a human symbol if you have not set one up).
+    2. Tap "Add another account" to add your Google account.
+    3. You will be prompted to enter your email address and password, so enter your ECCS Cloud Email email address and password that you set up above.
+
+### Q. I forgot my password / I cannot change my password. What can I do?
+
+* See [ECCS Cloud Email – “Initial Setup Procedures”](/en/google/#initial-setup-procedures), and follow the instructions in “Task: Set your password and email address”.
+
+### Q. What is the difference among `10-digits-ID@utac`, `10-digits-ID@g.ecc`, and `xxxx@g.ecc` (email address named after your choice) email addresses?
+* `10-digits-ID@utac.u-tokyo.ac.jp`(e.g. `1234567890@utac.u-tokyo.ac.jp`)
+  * Although it resembles an email address, it cannot be used for sending emails. However, the system may still send notifications or other messages to this address. For instance, regarding Zoom or UTokyo Slack, system-generated emails sent to this address can be received in each individual’s mailbox of ECCS Cloud Email.
+  * You might use your UTokyo Account as your username when logging into the system. The 10-digit number preceding the `@` symbol is known as the common ID. Some system require you to enter only this common ID when logging in.
```

Comment:
```suggestion
  * This may be used as your user name when you sign in to the system with your UTokyo Account. The 10-digit number before @ is called the common ID, and some systems require only this common ID to sign in.
```